### ISSUE TITLE
Configureable Product Page

### ISSUE THREAD

**Issue by krebbi** [VividWeb/vivid_store#95]
Just asking myself what would be the best way to have a fully configureable product page. 
one option is to make the vivid_product block editable and the other option would be splitting the product into several blocks and work with a page template. 
the last option would be more flexible in terms of adding more features like galleries or whatever. 

what do you think and do you need some help here?

**Comment by brucewyne** [comment]
Personally I think it's pretty flexible as is, at least for the content currently available for the product. You can easily go to the page type, select a different custom template, select the product information to show...

I'm open to hearing additional thoughts though.

**Comment by krebbi** [comment]
/dashboard/pages/types/output/ of the Page Type "Product Page" is not configureable so it's more a pure coding thing to adjust the layout & design of the product page. 
Or maybe have the template choosable on "Detail Page" in /dashboard/store/products/edit/ in the first place?

**Comment by brucewyne** [comment]
You are correct that it's somewhat of a code thing. It's not quite "configurable" as much as it's you can make any block template you'd like and select the default template output here. 

I do think it might be a good idea to explore possibly choosing the block template though at the product level. Could be cool.

**Comment by brucewyne** [comment]
Also - just realized we're not allowing you to select which Page Template you can use at the creation of the Product/Page #110

**Comment by brucewyne** [comment]
I don't want the product block itself configuring the design/layout. I think that makes the view.php way too messy and complicated. Which is what 5.6 eCommerce suffers from. I think the thing to do here is
- Provide a couple additional Product block templates
- When adding/editing a block, be able to set the Page Template and Block Template from that page.
